diff -up macemu-master.foo/BasiliskII/src/Unix/configure.ac~ macemu-master.foo/BasiliskII/src/Unix/configure.ac
--- macemu-master.foo/BasiliskII/src/Unix/configure.ac~	2013-03-04 19:14:20.000000000 +0100
+++ macemu-master.foo/BasiliskII/src/Unix/configure.ac	2013-03-10 16:34:54.553217381 +0100
@@ -250,7 +250,7 @@ else
 fi
 
 dnl We need X11, if not using SDL.
-if [[ "x$WANT_SDL" = "xno" ]]; then
+if [[ "x$WANT_SDL_VIDEO" = "xno" ]]; then
   AC_PATH_XTRA
   if [[ "x$no_x" = "xyes" ]]; then
     AC_MSG_ERROR([You need X11 to run Basilisk II.])
